app/assets/stylesheets/application.css

```diff
@@ -13,3 +13,7 @@
 20px 0}.mod-header li:last-child{margin-right:0}}.mod-outline{padding:1em 0 1em 4em}.mod-outline ul,.mod-outline ol{list-style:square;margin:0.5em 0em 0.5em 2em}.mod-outline ul li,.mod-outline ol li{margin:0.5em 0em}.mod-outline .layout-inner-wrap{min-width:90%}.mod-outline h1,.mod-outline h2,.mod-outline h3,.mod-outline h4,.mod-outline h5,.mod-outline p{margin:0 auto;line-height:27px}.mod-outline h1,.mod-outline h2,.mod-outline h3,.mod-outline h4,.mod-outline h5{margin:1em 0em 0em 0em}.mod-outline h1{margin-bottom:1em;font-size:2.8em;font-weight:200;color:#05c2d1}.mod-outline h2{font-size:2.2em;line-height:2.2em;text-decoration:underline}.mod-outline h3{font-size:1.8em;color:#05c2d1;line-height:1.8em}.mod-outline h4{font-size:1.6em;line-height:1.6em;font-weight:200}.mod-outline h5{font-size:1.4em;line-height:1.4em}.mod-outline h5 .date{color:#05c2d1}.mod-outline p,.mod-outline li{font-size:1.2em;margin-bottom:1em;line-height:1.5em}.col-article{float:left;width:75%}@media all and (max-width: 960px){.col-article{width:100%}}.col-sidebar{float:right;width:22.5%;text-align:left;padding-left:2.5%}@media all and (max-width: 960px){.col-sidebar{display:none}}.outline_list .month{position:relative;clear:both;overflow:visible;margin-bottom:75px}.outline_list .month .monthname{vertical-align:top;position:absolute;top:0;left:0;margin:0px}.outline_list .month .monthname a{color:rgba(5,194,209,0.1);margin:0px;font-weight:bold;font-size:150px;text-decoration:none}.outline_list .month .week{display:inline-block;margin-top:0px;margin-left:50px;margin-bottom:0px;position:relative;vertical-align:top;height:165px;width:75px}.outline_list .month .week ul.weekdays{list-style-type:none;margin:0px}.outline_list .month .week .weeknumber{-webkit-transform:rotate(-90deg);-moz-transform:rotate(-90deg);-o-transform:rotate(-90deg);-ms-transform:rotate(-90deg);transform:rotate(-90deg);z-index:-1;position:absolute;width:1em;height:1em;left:-45px;bottom:0}.outline_list .month .week .weeknumber a{color:#dfdfdf;font-weight:bold;font-size:45px;text-decoration:none}.mod-quote{padding:150px 0 200px;font-size:18px;line-height:1.22222;color:#fff;background:#025960 center center no-repeat;background-size:cover}.mod-quote.right{text-align:right}.mod-quote.right blockquote{margin:0 50px 0 0}.mod-quote blockquote{max-width:450px;display:inline-block;margin:0 0 0 110px}.mod-quote footer{font-size:14px;font-style:italic;line-height:1.57142}.mod-quote footer b{font-weight:700}.mod-quote a{color:#fff}@media (max-width: 760px){.mod-quote{padding:100px 0}.mod-quote.right{text-align:left}.mod-quote blockquote{margin:0}}.mod-text{padding:55px 0 100px}.mod-text h2{margin-bottom:65px;font-size:48px;font-weight:200;text-align:center}.mod-text h3{margin:39.99996px 0 10px;font-size:32px;font-weight:200;letter-spacing:0.01em}.mod-text ul,.mod-text ol{padding-left:20px;line-height:1.42857}.mod-text p,.mod-text li{margin-bottom:19.99998px}.mod-text em{font-style:italic}.mod-text strong{font-weight:800}
```

Comment:
The problem I have with this is that there is a lot of extra CSS in there that we probably won't use and don't need. I will go through this to find the relevant styling because I think this is just too much. thoughts?